strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/security/ldap/support/LdapGroupSearchXmlHolder.java

```diff
@@ -0,0 +1,33 @@
+package org.carlspring.strongbox.controllers.security.ldap.support;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+@XmlRootElement(name = "groupSearch")
+@XmlAccessorType(XmlAccessType.NONE)
+public class LdapGroupSearchXmlHolder
+{
+
+    @XmlElement(name = "searchBase")
+    private String searchBase;
+
+    @XmlElement(name = "searchFilter")
+    private String searchFilter;
+
+    LdapGroupSearchXmlHolder searchBase(String searchBase)
+    {
+        this.searchBase = searchBase;
+        return this;
+    }
+
+    LdapGroupSearchXmlHolder searchFilter(String searchFilter)
+    {
+        this.searchFilter = searchFilter;
+        return this;
+    }
+}
```

Comment:
Cosmetic: Could you please make sure any added/updated files end with a new line? While it's usually fine for classes, sometimes it causes problems with resource files, so -- just to be safe -- could you please keep an eye on these? Thanks! :)